### PR TITLE
Adding tests to about landing page.

### PR DESCRIPTION
**Pull request by KimberlyMunoz**
This is a duplicate of #785.

### CODE REVIEW

File: test/browser_tests/spec_suites/shared/shared_about-us.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  it( 'should properly load in a browser', function() {
+    expect( page.pageTitle() ).toBe( 'About Us' );
+  } );
+
+  it( 'should load items in the sidebar', function() {
+    expect( page.activityBlock ).toExist;
+  } );
+
+  it( 'should have Blog & Newspaper section in the Activity feed', function() {
+    var firstIcon = page.feedIcons.first();
```
Comment (jimmynotjim) [inline test/browser_tests/spec_suites/shared/shared_about-us.js]: We've started moving these to the page objects file to keep things separated.

File: test/browser_tests/spec_suites/shared/shared_about-us.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    var secondIcon = page.feedIcons.last();
+
+    expect(
+      firstIcon.getAttribute( 'class' )
+    ).toMatch( 'cf-icon cf-icon-speech-bubble' );
+
+    expect(
+      secondIcon.getAttribute( 'class' )
+    ).toContain( 'cf-icon cf-icon-newspaper' );
+
```
Comment (jimmynotjim) [inline test/browser_tests/spec_suites/shared/shared_about-us.js]: Extra line break here and L:36

File: test/browser_tests/page_objects/page_about-us.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+function AboutUs() {
+  this.get = function() {
+    browser.get( '/about-us/' );
+  };
+
+  this.pageTitle = function() { return browser.getTitle(); };
+  this.activityBlock = element.all( by.css( 'aside.activity' ) );
```
Comment (jimmynotjim) [inline test/browser_tests/page_objects/page_about-us.js]: There's no need for the `aside` in either of these is there?

File: test/browser_tests/spec_suites/shared/shared_about-us.js (hunk 1 above)
Comment (KimberlyMunoz) [inline test/browser_tests/spec_suites/shared/shared_about-us.js]: That's L24-25, right?

Comment (jimmynotjim) [inline test/browser_tests/spec_suites/shared/shared_about-us.js]: Correct

File: test/browser_tests/page_objects/page_about-us.js (hunk 3 above)
Comment (KimberlyMunoz) [inline test/browser_tests/page_objects/page_about-us.js]: I think you're right. I'll run it without the aside to confirm, but that's a good catch.

Comment (KimberlyMunoz) [inline test/browser_tests/page_objects/page_about-us.js]: And fixed in 6df98e6

File: test/browser_tests/spec_suites/shared/shared_about-us.js (hunk 2 above)
Comment (KimberlyMunoz) [inline test/browser_tests/spec_suites/shared/shared_about-us.js]: Fixed in 6df98e6

File: test/browser_tests/spec_suites/shared/shared_about-us.js (hunk 1 above)
Comment (KimberlyMunoz) [inline test/browser_tests/spec_suites/shared/shared_about-us.js]: Fixed in 6df98e6

**Comment by jimmynotjim** [comment]
:+1:

**Comment by imuchnik** [comment]
Truly sincere :+1: 
:fireworks:

**Pull request merged**